### PR TITLE
Update configs

### PR DESCRIPTION
**Pull request by diogomatsubara**
- fix formatting (autoformatted by VSCode)
- add customRuleset('main') to zenoh-pico, zenoh-cpp and ci repositories
- add `dependabot_security_updates_enabled: true` to all active `eclipse-zenoh/zenoh-*` repositories

### CODE REVIEW

**Review by netomi — DISMISSED** [review]

File: otterdog/eclipse-zenoh.jsonnet [hunk 1]
```diff
@@ -281,173 +289,183 @@ orgs.newOrg('eclipse-zenoh') {
...
     },
     orgs.newRepo('zenoh-plugin-dds') {
       allow_auto_merge: true,
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
-      description: "A zenoh plug-in that allows to transparently route DDS data.  This plugin can be used by DDS applications, such as ROS2 robotic applications and others, to leverage the zenoh for geographical routing or for better scaling discovery.",
-      homepage: "",
+      dependabot_security_updates_enabled: true,
+      description: 'A zenoh plug-in that allows to transparently route DDS data.  This plugin can be used by DDS applications, such as ROS2 robotic applications and others, to leverage the zenoh for geographical routing or for better scaling discovery.',
```
Comment (kydos) [inline otterdog/eclipse-zenoh.jsonnet]: For robotics application we should point to the ROS2 plugin. Can you please ask @JEnoch to update the text and add a link to the proper repo?

File: otterdog/eclipse-zenoh.jsonnet [hunk 2]
```diff
@@ -281,173 +289,183 @@ orgs.newOrg('eclipse-zenoh') {
...
     },
     orgs.newRepo('zenoh-plugin-ros2dds') {
       allow_auto_merge: true,
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
-      description: "A Zenoh plug-in for ROS2 with a DDS RMW.",
-      homepage: "https://zenoh.io",
+      dependabot_security_updates_enabled: true,
+      description: 'A Zenoh plug-in for ROS2 with a DDS RMW.',
```
Comment (kydos) [inline otterdog/eclipse-zenoh.jsonnet]: We should point to the RMW alternate report to explain the advantages of using Zenoh ROS2 plugin.  @JEnoch should  update this.

**Review by kydos — COMMENTED** [review]
Please take a look at the comments, once addressed this will be ready to be merged.

File: otterdog/eclipse-zenoh.jsonnet (hunk 1 above)
Comment (diogomatsubara) [reply otterdog/eclipse-zenoh.jsonnet]: How about:
```
A zenoh plug-in that allows to transparently route DDS data.  This plugin can be used by DDS applications to leverage zenoh for geographical routing or for better scaling discovery. For ROS2 robotic applications, use https://github.com/eclipse-zenoh/zenoh-plugin-ros2dds
```

File: otterdog/eclipse-zenoh.jsonnet (hunk 2 above)
Comment (diogomatsubara) [reply otterdog/eclipse-zenoh.jsonnet]: You mean this post: https://discourse.ros.org/t/ros-2-alternative-middleware-report/ ? How about:

```
A Zenoh plug-in for ROS2 with a DDS RMW. See https://discourse.ros.org/t/ros-2-alternative-middleware-report/  for the advantages of using this plugin over other DDS RMW implementations.
```

**Review by netomi — DISMISSED** [review]
LGTM

**Comment by netomi** [comment]
so from my POV this PR looks good and it integrates some changes that we would like to see. As the PR comes from an external contributor an approval from a PL is needed.

**Comment by kydos** [comment]
Looks good.

**Review by kydos — APPROVED** [review]
Looks good.

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
ty

**Pull request merged**